importexport/musicxml/importmxmlpass2.cpp

```diff
@@ -2580,16 +2586,22 @@ void MusicXMLParserDirection::direction(const QString& partId,
                         t->setFrameType(FrameType::SQUARE);
                         t->setFrameRound(0);
                         }
+
+                  QString wordsPlacement = placement();
+                  // Case-based defaults
+                  if (wordsPlacement.isEmpty())
+                        if (isVocalStaff) wordsPlacement = "above";
+                        else if (isExpressionText) wordsPlacement = "below";
```

Comment:
Here a warning reg. ambigues `else` is generated and additional curly braces are suggested.